x-pack/plugins/fleet/server/errors/handlers.ts

```diff
@@ -84,23 +63,6 @@ const getHTTPResponseCode = (error: IngestManagerError): number => {
 
 export function ingestErrorToResponseOptions(error: IngestErrorHandlerParams['error']) {
   const logger = appContextService.getLogger();
-  if (isLegacyESClientError(error)) {
```

Comment:
I am wondering if we can intercept errors from the new ES client, here. I think this handler was mean to get better error message instead of just `elasticsearch client returned a 400` and I think we loose that capabilities when we migrated to the new ES client.

Reply:
It seems like the new ES client does not expose specifics in this way. Looking at https://github.com/elastic/kibana/pull/107135 it seems the security team was doing something similar and had to simply abandon their intercepting of legacy ES errors. Do you have any insight on where to look to maybe continue with this type of intercepting logic?

Reply:
I am not sure who we can do this, I think it's okay to just removing it for now, as it will match the current feature we have and we can always improve later